app/__init__.py

```diff
@@ -202,7 +202,7 @@ def add_blueprints(application):
 def setup_secure_cookies(application):
     application.secret_key = settings.EQ_SECRET_KEY
     application.permanent_session_lifetime = timedelta(
-        seconds=settings.EQ_SESSION_TIMEOUT_SECONDS)
+        seconds=settings.EQ_SESSION_TIMEOUT_SECONDS + settings.EQ_SESSION_TIMEOUT_GRACE_PERIOD_SECONDS)
```

Comment:
That sets the time that gets passed to the front end but doesn't modify the server-side timeout of the session